gitlab/v4/objects/reviewers.py

```diff
@@ -0,0 +1,17 @@
+from gitlab.base import RESTManager, RESTObject
+from gitlab.mixins import ListMixin
+
+__all__ = [
+    "ProjectMergeRequestReviewers",
+    "ProjectMergeRequestReviewersManager",
+]
+
+
+class ProjectMergeRequestReviewers(RESTObject):
+    pass
+
+
+class ProjectMergeRequestReviewersManager(ListMixin, RESTManager):
+    _path = "/projects/{project_id}/merge_requests/{mr_iid}/reviewers"
+    _obj_cls = ProjectMergeRequestReviewers
+    _from_parent_attrs = {"project_id": "project_id", "mr_iid": "iid"}
```

Comment:
Just one more nit @kesmik. We tend to use the singular for our objects/managers, so maybe like this:

```suggestion
class ProjectMergeRequestReviewerDetail(RESTObject):
    pass


class ProjectMergeRequestReviewerDetailManager(ListMixin, RESTManager):
    _path = "/projects/{project_id}/merge_requests/{mr_iid}/reviewers"
    _obj_cls = ProjectMergeRequestReviewers
    _from_parent_attrs = {"project_id": "project_id", "mr_iid": "iid"}
```

Reply:
Hello, @nejch i have reworked as you suggested, one thing that api functional tests failed. I can not reproduce that on my local setup (at least cosnantly), should I worry about that? (it was same with my previous commit, but rerun of piepile helped_

Reply:
@kesmik our functional tests are a bit flaky because of resource limitations on GitHub runners, sorry about that. I've restarted the failing job, thanks for your patience! LGTM otherwise:)